bundler.d/test.rb

```diff
@@ -1,5 +1,5 @@
 group :test do
-  gem 'mocha', '~> 1.1'
+  gem 'mocha', '~> 1.3.0'
```

Comment:
why not `~> 1.3`?

Reply:
it was `~> 1.1` and it didn't save us the trouble. Apparently, they make changes that can break us in Y versions.

Reply:
Maybe I'm wrong, but IMHO version freeze should make the system resilient to dependency changes, but in most unrestricting way possible. In our case, we are using features that bound to change in Y versions. Hence upgrades to the next Y should be controlled, and not sporadic.